### DISCUSSION TITLE
Any plan to support streaming for whisper cloud.

### DISCUSSION THREAD

**Discussion by eltoob** [deepgram/community#931] [General help] [answered]
Hi,
I am trying to use whisper for some languages (or custom models) and I wanted to know if there is any plan to get whisper to stream rather than just using it in batch mode?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by deepgram-community** [comment, marked as answer]
hey there. there are no plans to expand our use of whisper. what languages are you finding you need that we don't cover already?
> luke

**Reply by eltoob** [reply]
hebrew
